tests/ci/README.md

```diff
@@ -79,4 +79,15 @@ Runs all tests with:
 CI Tool|Compiler|CPU platform|OS
 ------------ | -------------| -------------|-------------
 CodeBuild|clang 9.0.0|x86-64|Ubuntu 19.10
-CodeBuild|clang 9.0.0|aarch64|ubuntu 19.10
\ No newline at end of file
+CodeBuild|clang 9.0.0|aarch64|ubuntu 19.10
+
+### Valgrind tests
+
+The following Valgrind tests are run for a subset of targets in `utils/all_tests.json` using the debug build of AWS-LC:
+
+CI Tool|Compiler|CPU platform|OS| memcheck 
+------------ | -------------| -------------|-------------|-------------
+CodeBuild|gcc 7.3.1|x86-64|AL2 | X
+(Disabled) CodeBuild|gcc 7.3.1|aarch64|AL2 | X
```

Comment:
You mention that this target is disabled, but looking at the CodeBuild spec, it appears that it should still run the Valgrind tests. Am I just missing something here?

Reply:
Yeah, the codebuild spec it not the one used in the CI. It is "hard-coded" in the service and must be updated manually.

Reply:
But once we redeploy the CI using CDK, won't it enable this CodeBuild target for aarch64? Or do we not plan to do that?